File: psup assignment/java/q8.java
import java.util.*;
class CompareNumbers {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter first number: ");
        int num1 = input.nextInt();
        System.out.print("Enter second number: ");
        int num2 = input.nextInt();
        if (num1 > num2) {
            System.out.println("The larger number is: " + num1);
        } else {
            System.out.println("The larger number is: " + num2);
        }
        input.close();
    }
}

File: psup assignment/cpp/q8.cpp
#include <iostream>
#include <string>
#include <cctype>
int main() {
    int num1, num2;
    std::cout << "\n--- Program 8: Compare Two Numbers ---\n";
    std::cout << "Enter first number: ";
    std::cin >> num1;
    std::cout << "Enter second number: ";
    std::cin >> num2;
    if (num1 > num2) {
        std::cout << "The larger number is: " << num1 << std::endl;
    } else {
        std::cout << "The larger number is: " << num2 << std::endl;
    }
    return 0;
}